docs/user-guide-active-vs-on-demand.md

```diff
@@ -0,0 +1,20 @@
+# Active vs On-Demand Mode
+
+In this project we provide 2 handlers (one per each CRD). These handlers can be
+invoked in on on-demand or active mode.
+
+## On-Demand
+
+When launched in on-demand (client/one-shot) mode, the handler would run on
+selected nodes and handle `NodeNetworkState`/`NodeNetworkConfigurationPolicy`
+objects available at the moment. Once the handling is done, it would stop.
+
+This mode could be used by an external user, e.g.
+[Machine Config Operator](https://github.com/openshift/machine-config-operator).
+
+## Active
+
+When launched in active mode, the handler would be distributed across cluster as
+an daemon set and keep listening for
```

Comment:
an >> a

Drop the `keep`  + listening > listen

Reply:
Done